indigo/distribution.yaml

```diff
@@ -1451,7 +1451,7 @@ repositories:
       tags:
         release: release/indigo/{package}/{version}
       url: https://github.com/asmodehn/catkin_pip-release.git
```

Comment:
@asmodehn Just checking: this is released from a different organization than the one you released in lunar. The source repository is also different. Are they both used for development and release or do the urls need to be updated to point to the same org ?